crates/ruma-events/src/poll/response.rs

```diff
@@ -93,7 +93,7 @@ impl SelectionsContentBlock {
     pub fn validate<'a>(
         &'a self,
         poll: &PollContentBlock,
-    ) -> Option<impl Iterator<Item = &'a str>> {
+    ) -> Option<impl Iterator<Item = &'a str> + use<'a>> {
```

Comment:
So this was part of the solution? I thought any mention of `'a` in the type would act as a `use<'a>` already, so this wouldn't be necessary. If it is, I would have expected `cargo fix --edition` to make this edit, which it didn't.